lib/assessment.md

```diff
@@ -41,7 +54,17 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+.big {
+}
+
+#contentPane {
+}
+
+.important .red {
```

Comment:
There are a couple of CSS selector combinations that look similarly but act very differently:
`.important.red`
=> Intersection of important and red (only things that are both)
`.important, .red (note the comma)`
=> Union of important and red (important things, red things, and things that are both red and important)
`.important .red`
=> Elements with the class red that are descendants (inside) of elements with the class important.